tornjak-backend/cmd/agent/agent.go

```diff
@@ -41,7 +41,7 @@ func main() {
 				Usage:       "Config file path for spire server",
 				Destination: &opt.genericOptions.configFile,
 				Required:    true,
-			},
+			}, // TODO new argument
```

Comment:
actually let's remove this line for now, and we can handle the parsing in a new pr!